### DISCUSSION TITLE
calling setCurrentTime causes controls to show while video is playing

### DISCUSSION THREAD

**Discussion by eballeste** [vimeo/player.js#1005] [Feature requests]
### Expected Behavior
Would love to only see the controls if user is interacting directly with the video while the video is playing. I don't want to completely hide the controls because I want users to be able to pause/play on hover but I also don't want the controls interfering with the video playback while using external controls to view different parts of the video. Seems like a UX bug more than a feature.

### Actual Behavior
Playing the video and using `setCurrentTime`, triggered by clicking on external buttons that correspond to different timestamps, will show the pause control button and in my case, this is the big button right in the center of the video. 

https://user-images.githubusercontent.com/6924108/105280796-9ccfcf80-5b78-11eb-9860-1b050fefa8d9.mov

### Steps to Reproduce
In the demo, play the video and use the `setCurrentTime` button. The controls will pop in and then fade out each time you press the button.

**Comment by bdougherty** [comment]
I can bring this up with the team, but the intent of this feature was so that the viewer would know that something changed.
